src/Octokit.Webhooks.AspNetCore/GitHubWebhookExtensions.cs

```diff
@@ -11,15 +11,20 @@
     using Microsoft.AspNetCore.Http;
     using Microsoft.AspNetCore.Routing;
     using Microsoft.Extensions.DependencyInjection;
+    using Microsoft.Extensions.Logging;
 
     public static class GitHubWebhookExtensions
     {
         public static void MapGitHubWebhooks(this IEndpointRouteBuilder endpoints, string path = "/api/github/webhooks", string secret = null!) =>
             endpoints.MapPost(path, async context =>
             {
+                var loggerFactory = context.RequestServices.GetRequiredService<ILoggerFactory>();
+                var logger = loggerFactory.CreateLogger("Octokit.Webhooks.AspNetCore");
```

Comment:
Why do you need a `LoggerFactory` here instead of just a `Logger`?

Reply:
`ILogger` can't be resolved from the service container by itself, you need to resolve it as `ILogger<T>`. `T` needs to be a non-static type, and this endpoint is defined in the static `GitHubWebhookExtensions` class, so it isn't a valid candidate.

I figured a named logger would be the best alternative, but I could change it to an `ILogger<T>` if you nominate a type to use that will be used as the logger's name.

Reply:
Okay, that makes sense. It looks like `Terrajobst.GitHubEvents` uses `ILogger<IGitHubEventProcessor>`[^1]. The equivalent class in this library is `WebhookEventProcessor`.

What do you think about using that here?

[^1]: https://github.com/terrajobst/Terrajobst.GitHubEvents/blob/cb86100c783373e198cefb1ed7e92526a44833b0/src/Terrajobst.GitHubEvents.AspNetCore/GitHubEventsExtensions.cs#L22

Reply:
Sure, I'm fine with using that instead. I'll change that shortly.